Library/HUMSlider.m

```diff
@@ -472,6 +472,35 @@ - (void)setDesaturatedColor:(UIColor *)desaturatedColor
     self.rightDesaturatedImageView.tintColor = _desaturatedColor;
 }
 
+- (void)setSaturatedColor:(UIColor *)saturatedColor forSide:(HUMSliderSide)side
+{
+    [self imageViewForSide:side saturated:YES].tintColor = saturatedColor;
+}
+
+- (UIColor*)saturatedColorForSide:(HUMSliderSide)side
+{
+    return [self imageViewForSide:side saturated:YES].tintColor;
+}
+
+- (void)setDesaturatedColor:(UIColor *)desaturatedColor forSide:(HUMSliderSide)side
+{
+    [self imageViewForSide:side saturated:NO].tintColor = desaturatedColor;
+}
+
+- (UIColor*)desaturatedColorForSide:(HUMSliderSide)side
+{
+    return [self imageViewForSide:side saturated:NO].tintColor;
+}
+
+- (UIImageView*)imageViewForSide:(HUMSliderSide)side saturated:(BOOL)saturated
+{
+    if (side == HUMSliderSideLeft) {
```

Comment:
I would recommend making this `if` statement dependent on `saturated` and then using a `switch` within the `if` statement for the `side` - generally I prefer to use `switch` statements for anything which is an enum since it can conceivably get more options in the future. 

Also I have an irrational hatred of ternary operators, so a nice side effect is it gets rid of those. :)